tests/test_config.py

```diff
@@ -0,0 +1,7 @@
+import retasc.validator.config as config
+
+
+def test_schema_version_exists():
```

Comment:
This does not seem very useful. It is obvious that the option is defined.

Reply:
completely agree, but I think we have 100% test coverage requirement. How would you test this file without being obvious?